### ISSUE TITLE
Unable to install Gryphon

### ISSUE THREAD

**Issue by long-brivvio** [vinivendra/homebrew-gryphon#4]
Hi, I could not install Gryphon even after switching back to xcode 13.2.1

Here's the bit of code from terminal 
```
admin@Mac-mini app-android % brew install vinivendra/gryphon/gryphon  
Running `brew update --preinstall`...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
Updated 2 formulae.

==> Downloading https://rubygems.org/downloads/xcodeproj-1.16.0.gem
Already downloaded: /Users/admin/Library/Caches/Homebrew/downloads/9e7f02841e532e6e0113019acaa6459ebb34d50c4f4f856356ef4d38cfa5e2d4--xcodeproj-1.16.0.gem
==> Downloading https://github.com/vinivendra/Gryphon/archive/v0.18.1.tar.gz
Already downloaded: /Users/admin/Library/Caches/Homebrew/downloads/8b629ab4c4d811f72d9a6169fa3846e116eb5a4879c49493075429d3ad8dddb5--Gryphon-0.18.1.tar.gz
==> Installing gryphon from vinivendra/gryphon
Warning: Your Xcode (13.2.1) is outdated.
Please update to Xcode 13.3 (or delete it).
Xcode can be updated from the App Store.

==> Downloading https://rubygems.org/downloads/xcodeproj-1.16.0.gem
Already downloaded: /Users/admin/Library/Caches/Homebrew/downloads/9e7f02841e532e6e0113019acaa6459ebb34d50c4f4f856356ef4d38cfa5e2d4--xcodeproj-1.16.0.gem
==> gem install --install-dir /opt/homebrew/Cellar/gryphon/0.18.1/libexec/vendor /Users/admin/Lib
==> swift build --disable-sandbox
Last 15 lines from /Users/admin/Library/Logs/Homebrew/gryphon/02.swift:
           ^
Swift.MutableCollection:6:14: note: requirement 'subscript(_:)' declared here
    override subscript(bounds: Range<Self.Index>) -> Self.SubSequence { get set }
             ^
[127/138] Compiling GryphonLib AuxiliaryFileContents.swift
[128/138] Compiling GryphonLib Compiler.swift
[129/138] Compiling GryphonLib Driver.swift
[130/138] Compiling GryphonLib LibraryTranspilationPass.swift
[131/138] Compiling GryphonLib PrintableAsTree.swift
[132/138] Compiling GryphonLib RubyScriptContents.swift
[133/138] Compiling GryphonLib Shell.swift
[134/138] Compiling GryphonLib TranspilationPass.swift
[135/138] Compiling GryphonLib Utilities.swift
[136/138] Compiling GryphonLib SourceFile.swift
[137/138] Compiling GryphonLib SwiftSyntaxDecoder.swift

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/vinivendra/homebrew-gryphon/issues


Error: Your Xcode (13.2.1) is outdated.
Please update to Xcode 13.3 (or delete it).
Xcode can be updated from the App Store.
```

Thanks!

**Comment by long-brivvio** [comment]
Hi, 
Just an update on the issue.
Using mint to install Gryphon did work but there's another issue when I set up the project.
```
admin@Mac-mini app copy % mint run gryphon init app.xcodeproj/ --target='app' --verbose
🌱 Finding latest version of Gryphon
🌱 Running gryphon v0.18.1...
ℹ️  Gryphon version 0.18.1
ℹ️  SwiftSyntax version 5.5
🧑‍💻  Checking Xcode arguments...
	ℹ️  Using target app.
	ℹ️  Using Xcode project app.xcodeproj.
✅  Done checking Xcode arguments.
🧑‍💻  Initializing...
	ℹ️  Generating xcode files
	ℹ️  Gryphon version 0.18.1
	ℹ️  SwiftSyntax version 5.5
	🧑‍💻  Checking Xcode arguments...
		ℹ️  Using target app.
		ℹ️  Using Xcode project app.xcodeproj.
	✅  Done checking Xcode arguments.
	🧑‍💻  Creating iOS compilation files...
		🛠  xcodebuild -UseModernBuildSystem=NO -project app.xcodeproj -target app -dry-run
Error running xcodebuild:
Command line invocation:
    /Users/admin/Downloads/Xcode.app/Contents/Developer/usr/bin/xcodebuild -UseModernBuildSystem=NO -project app.xcodeproj -target app -dry-run

User defaults from command line:
    IDEPackageSupportUseBuiltinSCM = YES
    UseModernBuildSystem = NO

Resolve Package Graph
Packages are not supported when using the legacy build system.
Fetching from https://github.com/kirualex/SwiftyGif.git (cached)

Fetching from https://github.com/mixpanel/mixpanel-swift (cached)

Fetching from https://github.com/firebase/firebase-ios-sdk.git (cached)

Fetching from https://github.com/iwasrobbed/Down.git (cached)

Fetching from https://github.com/firebase/leveldb.git (cached)

Fetching from https://github.com/firebase/abseil-cpp-SwiftPM.git (cached)

Fetching from https://github.com/grpc/grpc-ios.git (cached)

Fetching from https://github.com/firebase/nanopb.git (cached)

Fetching from https://github.com/google/gtm-session-fetcher.git (cached)

Fetching from https://github.com/google/GoogleUtilities.git (cached)

Fetching from https://github.com/google/GoogleAppMeasurement.git (cached)

Fetching from https://github.com/google/GoogleDataTransport.git (cached)

Fetching from https://github.com/apple/swift-protobuf.git (cached)

Fetching from https://github.com/google/promises.git (cached)

Cloning local copy of package ‘Down’

Checking out 0.11.0 of package ‘Down’

Cloning local copy of package ‘GoogleDataTransport’

Checking out 9.1.4 of package ‘GoogleDataTransport’

Cloning local copy of package ‘firebase-ios-sdk’

Checking out 8.15.0 of package ‘firebase-ios-sdk’

Cloning local copy of package ‘swift-protobuf’

Checking out 1.19.0 of package ‘swift-protobuf’

Cloning local copy of package ‘nanopb’

Checking out 2.30908.0 of package ‘nanopb’

Cloning local copy of package ‘SwiftyGif’

Checking out master of package ‘SwiftyGif’

Cloning local copy of package ‘mixpanel-swift’

Checking out master of package ‘mixpanel-swift’

Cloning local copy of package ‘leveldb’

Checking out 1.22.2 of package ‘leveldb’

Cloning local copy of package ‘promises’

Checking out 2.1.0 of package ‘promises’

Cloning local copy of package ‘gtm-session-fetcher’

Checking out 1.7.2 of package ‘gtm-session-fetcher’

Cloning local copy of package ‘GoogleUtilities’

Checking out 7.7.1 of package ‘GoogleUtilities’

Cloning local copy of package ‘grpc-ios’

Checking out 1.44.0-grpc of package ‘grpc-ios’

Cloning local copy of package ‘GoogleAppMeasurement’

Checking out 8.15.0 of package ‘GoogleAppMeasurement’

Cloning local copy of package ‘abseil-cpp-SwiftPM’

Checking out 0.20220203.1 of package ‘abseil-cpp-SwiftPM’

Updating from https://github.com/kirualex/SwiftyGif.git

Updating from https://github.com/mixpanel/mixpanel-swift

Updating from https://github.com/iwasrobbed/Down.git

Updating from https://github.com/firebase/firebase-ios-sdk.git

Updating from https://github.com/firebase/leveldb.git

Updating from https://github.com/grpc/grpc-ios.git

Updating from https://github.com/firebase/abseil-cpp-SwiftPM.git

Updating from https://github.com/firebase/nanopb.git

Updating from https://github.com/google/gtm-session-fetcher.git

Updating from https://github.com/google/GoogleUtilities.git

Updating from https://github.com/google/GoogleDataTransport.git

Updating from https://github.com/google/GoogleAppMeasurement.git

Updating from https://github.com/apple/swift-protobuf.git

Updating from https://github.com/google/promises.git

Fetching from https://github.com/firebase/boringssl-SwiftPM.git (cached)

Cloning local copy of package ‘boringssl-SwiftPM’

Checking out 0.9.0 of package ‘boringssl-SwiftPM’

ignoring broken symlink /Users/admin/Library/Developer/Xcode/DerivedData/app-ezxnmrrmpiirwpdkqekygxivazup/SourcePackages/checkouts/grpc-ios/native_src/spm-core-include/grpcignoring broken symlink /Users/admin/Library/Developer/Xcode/DerivedData/app-ezxnmrrmpiirwpdkqekygxivazup/SourcePackages/checkouts/grpc-ios/native_src/spm-core-include/grpc
Resolved source packages:
  GoogleAppMeasurement: https://github.com/google/GoogleAppMeasurement.git @ 8.15.0
  Mixpanel: https://github.com/mixpanel/mixpanel-swift @ master
  Promises: https://github.com/google/promises.git @ 2.1.0
  GoogleUtilities: https://github.com/google/GoogleUtilities.git @ 7.7.1
  Firebase: https://github.com/firebase/firebase-ios-sdk.git @ 8.15.0
  SwiftProtobuf: https://github.com/apple/swift-protobuf.git @ 1.19.0
  SwiftyGif: https://github.com/kirualex/SwiftyGif.git @ master
  gRPC: https://github.com/grpc/grpc-ios.git @ 1.44.0-grpc
  abseil: https://github.com/firebase/abseil-cpp-SwiftPM.git @ 0.20220203.1
  leveldb: https://github.com/firebase/leveldb.git @ 1.22.2
  Down: https://github.com/iwasrobbed/Down.git @ 0.11.0
  BoringSSL-GRPC: https://github.com/firebase/boringssl-SwiftPM.git @ 0.9.0
  GoogleDataTransport: https://github.com/google/GoogleDataTransport.git @ 9.1.4
  nanopb: https://github.com/firebase/nanopb.git @ 2.30908.0
  GTMSessionFetcher: https://github.com/google/gtm-session-fetcher.git @ 1.7.2

warning: unable to restore checkouts state: unable to restore state from /Users/admin/Library/Developer/Xcode/DerivedData/app-ezxnmrrmpiirwpdkqekygxivazup/SourcePackages/repositories/checkouts-state.json; missingKey("status")
objc[2335]: Class AMSupportURLConnectionDelegate is implemented in both /usr/lib/libauthinstall.dylib (0x1f15f2b90) and /Library/Apple/System/Library/PrivateFrameworks/MobileDevice.framework/Versions/A/MobileDevice (0x1039e82c8). One of the two will be used. Which one is undefined.
objc[2335]: Class AMSupportURLSession is implemented in both /usr/lib/libauthinstall.dylib (0x1f15f2be0) and /Library/Apple/System/Library/PrivateFrameworks/MobileDevice.framework/Versions/A/MobileDevice (0x1039e8318). One of the two will be used. Which one is undefined.
xcodebuild: error: Could not resolve package dependencies:
  Packages are not supported when using the legacy build system.
```

**Comment by vinivendra** [comment]
Hi @long-brivvio, this seems to be the same issue as #3, which was fixed in Gryphon version `0.19` with support for Swift 5.6. Please try it out and let me know if it still doesn't work.
